socialhome/users/tests/test_views.py

```diff
@@ -131,12 +138,12 @@ def test_get_json_context(self):
                 "profile": {
                     "id": profile.id,
                     "guid": profile.guid,
-                    "followersCount": 3,
-                    "followingCount": str(profile.following.count()),
+                    "followers_count": "0",
+                    "following_count": str(profile.following.count()),
                     "handle": profile.handle,
-                    "saferImageUrlLarge": profile.safer_image_url_large,
-                    "streamType": view.profile_stream_type,
-                    "pinnedContentExists": view.pinned_content_exists,
+                    "safer_image_url_large": profile.safer_image_url_large,
+                    "stream_type": view.profile_stream_type,
+                    "pinned_content_exists": view.pinned_content_exists,
```

Comment:
I don't think `view.pinned_content_exists` should exist any more since we use the serializer now?

Reply:
It's used on the Vue side.

Reply:
>  <b-button v-if="profile.has_pinned_content" 

I think you renamed it already in this PR :) Also, I was referring to the Django View. So [this](https://github.com/jaywink/socialhome/blob/6559f3ece9c7658ea797d2e01726e5c0ec24770d/socialhome/users/views.py#L106) and [this](https://github.com/jaywink/socialhome/blob/6559f3ece9c7658ea797d2e01726e5c0ec24770d/socialhome/users/views.py#L36) can be removed.

Reply:
Won't it break the jQuery front?